flink-connectors/flink-connector-filesystem/src/test/java/org/apache/flink/streaming/connectors/fs/StringWriterTest.java

```diff
@@ -32,7 +32,7 @@
 
 	@Test
 	public void testDuplicate() {
```

Comment:
Please also add a unit test (in this file?) to cover that the custom delimiter is actually used somehow.